scripts/apptester/apptester.py

```diff
@@ -364,7 +378,9 @@ def main():
     tester = AppTesterFramework(appconfig_file, kaanode, kaauser,
                                 args.rootpath, builddir, sandboxframe)
     if args.j:
-        tester.build_android_java_demo()
+        tester.build_java_android_demo()
```

Comment:
We need to add additional phase on Jenkins that will run the apptester with "-c" option.